app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1057,6 +1057,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = ""
     )
 
+    var overridePushServer by stringPreference(
```

Comment:
Why use a string pref instead of a boolean?
```suggestion
    var overridePushServer by booleanPreference(
```

Reply:
It stores the overriden URL.  Maybe I should have called it `overredPushServerUrl`, but I just tried to match the existing `overrideSyncTokenServer` pref.

Reply:
Ah, that's fair.